<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>fitVariableEditor</class>
 <widget class="QWidget" name="fitVariableEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>319</width>
    <height>23</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLineEdit" name="variableName"/>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>=</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="variableValue"/>
   </item>
   <item>
    <widget class="QCheckBox" name="tFactor">
     <property name="text">
      <string>t-Factor</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDoubleSpinBox" name="confidenceInterval">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="decimals">
      <number>4</number>
     </property>
     <property name="minimum">
      <double>0.000100000000000</double>
     </property>
     <property name="maximum">
      <double>0.999900000000000</double>
     </property>
     <property name="singleStep">
      <double>0.025000000000000</double>
     </property>
     <property name="value">
      <double>0.900000000000000</double>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>tFactor</sender>
   <signal>toggled(bool)</signal>
   <receiver>confidenceInterval</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>216</x>
     <y>9</y>
    </hint>
    <hint type="destinationlabel">
     <x>276</x>
     <y>10</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
